### ISSUE TITLE
min and max types not documented?

### ISSUE THREAD

**Issue by rdboisvert** [jmcnamara/excel-writer-xlsx#135]
Hi!

I'm using conditional_formatting and the 3-color scale.  The code below does exactly what I want but I couldn't find it in the documentation.  I hesitate to use an undocumented feature since it may change unexpectedly.  Is what I'm doing valid?  Is it already documented?  If not, can it be added?

Thanks,
Bob

```perl
$oSheet->conditional_formatting ($first_row, $first_col, $last_row, $last_col,  {
  type  => '3_color_scale',
  min_type  => 'min',
  mid_type  => 'percentile',
  max_type  => 'max',
  min_color => '#5A8AC6',
  mid_color => '#FFFFFF',
  max_color => '#F8696B',
});
```

**Comment by jmcnamara** [comment]
Hi,

The max and min types [are documented](http://search.cpan.org/dist/Excel-Writer-XLSX/lib/Excel/Writer/XLSX.pm#min_type,_mid_type,_max_type). No?

John

**Comment by rdboisvert** [comment]
Yes, the "min_type, mid_type, max_type" properties are documented but it doesn't indicate that their values could include "min" or "max" in addition to "num, percent, percentile, formula" which are shown.

I guess my question might be better stated as, Are "min" and "max" valid values for these properties?

**Comment by jmcnamara** [comment]
> I guess my question might be better stated as, Are "min" and "max" valid values for these properties?

The `min` and `max` values are valid but only for `min_type` and `max_type`, respectively. 

I'll update the docs to make this clear the update will be something like:

>The available min/mid/max types are:

    min        (for min_type only)
    num
    percent
    percentile
    formula
    max        (for max_type only)

**Comment by rdboisvert** [comment]
That works fine.

Thanks!

**Comment by jmcnamara** [comment]
Closed in version 0.86 on CPAN.

Thanks.
